providers/implementations/encode_decode/decode_msblob2key.c

```diff
@@ -93,6 +93,11 @@ static int msblob2key_decode(void *vctx, OSSL_CORE_BIO *cin, int selection,
     void *key = NULL;
     int ok = 0;
 
+    if (in == NULL) {
+        ERR_raise(ERR_LIB_PEM, ERR_R_MALLOC_FAILURE);
+        goto end;
```

Comment:
You can just return 0 here. all the calls after the end label are no-ops in this case. Also drop the ERR_raise() as BIO_new that is called inside the ossl_bio_new_from_core_bio() will raise an error anyway.